_opensearch/rest-api/update-mapping.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Update mapping
+parent: REST API reference
+nav_order: 6
+---
+
+# Update mapping
+
+If you want to update an index's mappings after creating the index, you can do so with the update mapping API operation.
```

Comment:
Any implications if I try to do this on an index with a whole bunch of data? Do the updates have to be additive rather than turning existing strings into integers (I assume this sort of request would error out)?

Basically, a bit more context what why I might do this and any gotchas to be aware of.